R/steps-rcmdcheck.R

```diff
@@ -13,7 +13,10 @@ TicStepWithPackageDeps <- R6Class(
       # (which might well be ahead of CRAN)
       # works very poorly with custom steps that are not aware
       # of this shadow library.
-      utils::update.packages(ask = FALSE)
+      installed_packages <- installed.packages()[, "Package"]
```

Comment:
Can you please disentangle?

```r
installed_packages <- ...
pkg_names <- ...
priority <- ...
pkg_to_update <- ...
remotes::update_packages()
```

Reply:
The many `installed.packages()` calls still make it hard to skim through the code: 

```r
inst <- installed.packages()
...
priority <- inst[, "Priority"]
pkg_to_update <- inst[!(priority %in% ...)]
...
```